testsuite/MDAnalysisTests/lib/test_cutil.py

```diff
@@ -46,6 +46,43 @@ def test_unique_int_1d(values):
     assert res.dtype == ref.dtype
 
 
+@pytest.mark.parametrize('arrays', (
+    [],  # empty array
+    [1, 1, 1, 1, ],  # all identical
+    [2, 3, 5, 7, ],  # unique, sorted
+    [5, 2, 7, 3, ],  # unique, unsorted
+    [1, 2, 2, 4, 4, 6, ],  # duplicates, sorted
+    [1, 2, 2, 6, 4, 4, ],  # duplicates, unsorted
+))
+def test_inverse_unique_contiguous_1d_array(arrays):
+    # Generate unique, unsorted arrays.
+    array = np.array(arrays, dtype=np.intp)
+    unique_array = unique_int_1d_unsorted(array)
+
+    inverse = inverse_unique_contiguous_1d_array(array, unique_array)
+    ref = array
+    res = unique_array[inverse]
+    assert_equal(res, ref)
+    assert type(res) == type(ref)
+    assert res.dtype == ref.dtype
+
+    # raise ValueError on non 1d inputs
+    with pytest.raises(ValueError):
+        # change array shape
+        array_2d = array.reshape((2, -1))
+        values, indices = np.unique(array_2d, return_index=True)
+        unique_array = array[np.sort(indices)]
+        inverse_unique_contiguous_1d_array(array_2d, unique_array)
```

Comment:
Could you please make this a separate test -- it doesn't need to run for each `parametrize` value.

Reply:
Just occured to me but what is the best practice for tests with a single set of parameters?
I used a `parameterize` here, but `test_ind2d` a little further down just hardcodes the values into the test.

Reply:
Yeah just do that if you only want one test :)